README.md

```diff
@@ -47,18 +47,16 @@ coverage:
   steps:
     - name: Install Go
       if: success()
-      uses: actions/setup-go@v2-beta
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.15.x
     - name: Checkout code
       uses: actions/checkout@v2
     - name: Calc coverage
-      run: |
-        export PATH=$PATH:$(go env GOPATH)/bin
+      run: go test -v -covermode=count -coverprofile=coverage.out
         go test -v -covermode=count -coverprofile=coverage.out
```

Comment:
.... this line need remove ...

Reply:
![image](https://user-images.githubusercontent.com/3842474/96352327-f1d0f500-10f4-11eb-8d2d-9402d1d4a245.png)

github limit, i can't span

Reply:
yes, done